.travis.yml

```diff
@@ -8,16 +21,27 @@ cache:
   - directories:
     - cache
 script:
-- export BUILD_DATE=$(date +%Y%m%d)
-- make EXTRA_CCFLAGS="-DBUILD_DATE='\"'$BUILD_DATE'\"'" all
-- cd bin/
-- file_name_float="nodemcu_float_${TRAVIS_TAG}.bin"
-- srec_cat -output ${file_name_float} -binary 0x00000.bin -binary -fill 0xff 0x00000 0x10000 0x10000.bin -binary -offset 0x10000
-- cd ../
-- make clean
-- make EXTRA_CCFLAGS="-DLUA_NUMBER_INTEGRAL -DBUILD_DATE='\"'$BUILD_DATE'\"'"
-- cd bin/
-- file_name_integer="nodemcu_integer_${TRAVIS_TAG}.bin"
-- srec_cat -output ${file_name_integer} -binary 0x00000.bin -binary -fill 0xff 0x00000 0x10000 0x10000.bin -binary -offset 0x10000
+- echo OS is $OS $TRAVIS_OS_NAME
+- if [ "$OS" = "linux" ]; then export BUILD_DATE=$(date +%Y%m%d); fi
```

Comment:
How about a single `if` for each OS and putting all commands into a script per OS?

Reply:
done as requested